src/generate/convert/mod.rs

```diff
@@ -84,6 +103,10 @@ pub fn convert_node(ast: &AST, imp: &mut Imports, state: &State) -> GenResult {
         Node::LeOp => Core::LeOp,
         Node::GeOp => Core::GeOp,
         Node::QuestionOp { .. } => convert_ty(ast, imp, state)?,
+        Node::BLShiftOp | Node::BRShiftOp => {
+            let msg = format!("Cannot have top level: {}", ast.node);
```

Comment:
Illegal top-level constructs seem to occur more often.
Should create helper function which just generates this error instead of doing it separately each time here.